src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html

```diff
@@ -8,29 +8,27 @@
                 <button class="btn btn-primary" data-bind="click: resume, visible: !liveClient()">
                     <i class="icon-play"></i> <span>Resume</span>
                 </button>
-                <button class="btn btn-default" data-bind="click: exportToFile">
-                    <i class="icon-export"></i><span>Export</span>
-                </button>
             </div>
             <div class="flex-horizontal flex-wrap margin-right">
-                <div>
-                    <input class="form-control" placeholder="Filter" data-bind="textInput: filter" title="Filter 'Custom Info' & 'URI' columns">
-                </div>
                 <div class="flex-horizontal">
                     <div class="btn-group">
                         <select id="visibleTypesSelector" size="5" multiple="multiple"
                                 data-bind="options: filteredTypeData, selectedOptions: selectedTypeNames, optionsText: 'propertyName', optionsValue: 'propertyName'"></select>
                     </div>
+                    <div id="trafficFilter">
+                        <input class="form-control" placeholder="Filter 'Custon Info' & 'URI' Columns" data-bind="textInput: filter" title="Filter 'Custom Info' & 'URI' Columns">
```

Comment:
no need for title no - we have placeholder

Reply:
done - title removed